frontend/ui/widget/keyboardlayoutdialog.lua

```diff
@@ -119,6 +121,11 @@ function KeyboardLayoutDialog:init()
         show_parent = self,
     }
 
+    local max_radio_button_container_height = math.floor(Screen:getHeight()*0.9
+                    - self.title_widget:getSize().h - self.title_bar:getSize().h
+                    - Size.span.vertical_large*4 - self.button_table:getSize().h)
```

Comment:
Wouldn't we normally write 0.9* and 4*? (Genuine question, not a request for change. Fine by me either way. ;-) )

Reply:
We have an assorment of both. A bit more `2*something` though, but a few `something*2` (including some `width = Size.span.vertical_large*2` by you in openwithdialog.lua :)
So, I guess it's not yet written in the stone of our style guide :)